# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332262784 18000
#      Tue Mar 20 11:59:44 2012 -0500
# Node ID 9328e5d72eb288f306a95050929426ad9a6724d2
# Parent  5e0bb4f071fc7edcdd26a2326e61301833f12177
cmake: add math library -lm

diff -r 5e0bb4f071fc -r 9328e5d72eb2 programs/CMakeLists.txt
--- a/programs/CMakeLists.txt	Tue Mar 20 11:59:43 2012 -0500
+++ b/programs/CMakeLists.txt	Tue Mar 20 11:59:44 2012 -0500
@@ -5,10 +5,10 @@ add_executable(mtest mtest.c io.c)
 add_executable(parmetis_prog parmetis.c io.c adaptgraph.c)
 set_target_properties(parmetis_prog PROPERTIES OUTPUT_NAME parmetis)
 add_executable(pometis pometis.c io.c)
 # Link to libparmetis.
 foreach(prog ptest mtest parmetis_prog pometis)
-  target_link_libraries(${prog} parmetis)
+  target_link_libraries(${prog} parmetis m)
 endforeach(prog)
 
 install(TARGETS ptest mtest parmetis_prog pometis
   RUNTIME DESTINATION bin)
